website/src/theme/Navbar/Layout/index.js

```diff
@@ -118,16 +118,14 @@ const telemetryInit = () => {
   } else {
     // remove Manage Cookie and separator in footer
     try {
+      // Remove both footer links and manage cookie ID
       removeItem("footer__links_" + manageCookieLabel);
       removeItem(manageCookieId);
-    } catch (error) {
-      // Ignore errors if elements don't exist
+    } catch (e) {
+      // Ignore if elements don't exist - this is expected behavior
+      // when cookies/elements have already been removed or don't exist
     }
   }
-  
-  if (WcpConsent && WcpConsent.siteConsent) {
-    setNonEssentialCookies(WcpConsent.siteConsent.getConsent());
-  }
```

Comment:
This seems unrelated, why was this removed?

Reply:
@hemarina Curious to see if this is an expected change for you.